python-package/securexgboost/core.py

```diff
@@ -1302,6 +1397,45 @@ def _add_client_key_with_certificate(self, certificate, data, data_len, signatur
         _LIB.add_client_key_with_certificate(certificate, cert_len, data, data_len, signature, sig_len)
 
 
+    def _get_enclave_symm_key(self):
+        """
+        Get enclave's symmetric key used to encrypt output common to all clients
+        """
+        if "current_user" in globals():
+            username = globals()["current_user"]
+        if username is None:
+            raise ValueError("Please set your username with the Set_user function or provide a username as an optional argument")
```

Comment:
I think this is deprecated. The user function is now called `securexgboost.init_user()`